src/graphql/main/queries.ts

```diff
@@ -17,7 +17,7 @@ const QueryType = new GT.Object({
     globals: GlobalsQuery,
     me: MeQuery,
     usernameAvailable: UsernameAvailableQuery,
-    userDefaultWalletId: UserDefaultWalletIdQuery,
+    userDefaultWalletId: AccountDefaultWalletIdQuery, // FIXME: migrate to AccountDefaultWalletId
```

Comment:
uhmm this is a bit confusing, imported from `"@graphql/root/query/user-default-wallet-id"` `userDefaultWalletId` as attribute but with `AccountDefaultWalletIdQuery` type name

Reply:
file renamed.

`userDefaultWalletId -> accountDefaultWalletId` will need a migration from the mobile app AFAIK

so my intent is to do it later in another PR because we will need both fields for some time during the transition